internal/metrics/telemetry/cluster_log_forwarder.go

```diff
@@ -0,0 +1,51 @@
+package telemetry
+
+import (
+	observabilityv1 "github.com/openshift/cluster-logging-operator/api/observability/v1"
+)
+
+func gatherForwarderInfo(forwarder *observabilityv1.ClusterLogForwarder) (pipelines uint, inputs, outputs []string) {
+	inputs = makeZeroStrings(len(forwarderInputTypes))
+	outputs = makeZeroStrings(len(forwarderOutputTypes))
+
+	outputMap := makeOutputMap(&forwarder.Spec)
+
+	activeInputNames := map[string]bool{}
+	activeOutputTypes := map[string]bool{}
+	for _, pipeline := range forwarder.Spec.Pipelines {
+		for _, inputName := range pipeline.InputRefs {
+			activeInputNames[inputName] = true
+		}
+
+		for _, outputName := range pipeline.OutputRefs {
+			output, found := outputMap[outputName]
+			if found {
+				activeOutputTypes[string(output.Type)] = true
+			}
+		}
+	}
+
+	pipelines = uint(len(forwarder.Spec.Pipelines))
+
+	for i, v := range forwarderInputTypes {
+		if activeInputNames[v] {
+			inputs[i] = "1"
```

Comment:
should we use "boolLabel" here?

Reply:
I just created `boolLabel` for the "inline usage", I don't think it would add any value here, just another function call and an unnecessary conditional (not sure, if that's optimized away). But I am quite neutral on this decision...

Should I make the "0" / "1" strings constants?